tools/download-vm.js

```diff
@@ -10,15 +10,16 @@ const fs = require('fs');
 const path = require('path');
 const mkdirp = require('mkdirp');
 
-const isoUrl = 'https://copy.sh/v86/images/linux3.iso';
+const isoUrl =
+    'https://github.com/humphd/next/releases/download/1.0/v86-linux.iso?raw=true';
```

Comment:
Can we somehow use `https://github.com/humphd/next/releases/latest`, which will be a 302 redirect to `https://github.com/humphd/next/releases/download/1.0/` (or whatever the latest is) to build this URL, so we don't have to change the code in here to get it to download the newest binaries?